sites/platform/static/files/fetch/examples/relationships/solr

```diff
@@ -1,18 +1,22 @@
 {
     "username": null,
-    "scheme": "solr",
-    "service": "solr86",
     "fragment": null,
     "ip": "169.254.68.119",
-    "hostname": "csjsvtdhmjrdre2uaoeim22xjy.solr86.service._.eu-3.platformsh.site",
-    "port": 8080,
     "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "solr.internal",
-    "rel": "solr",
     "path": "solr\/maincore",
     "query": [],
     "password": null,
+    "port": 8080,
+    "host_mapped": false,
+    "service": "solr86",
+    "hostname": "csjsvtdhmjrdre2uaoeim22xjy.solr86.service._.eu-3.platformsh.site",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.solr86.service._.eu-1.platformsh.site",